about.html

```diff
@@ -0,0 +1,145 @@
+      <li class="sessionbar px-4">Logout</li>
+    </ul>
+  </div>
+
+  <a id="logonav" href="#top-page" class="sm d-none">
+    <img src="./capstoneimgs/logo.png" alt="world-music-festival-logo"></a>
+  <div class=" secondary-nav">
+   
+    <ul class="d-flex justify-content-end px-4 mt-3">
+      <li class="sec-nav-item">
+        <a class="sec-nav-link px-4" href="index.html">Home</a>
+      </li>
+      <li class="sec-nav-item">
+        <a class="sec-nav-link px-4" href="#">Program</a>
+      </li>
+      <li class="sec-nav-item">
+        <a class="sec-nav-link px-4" href="#">Performers</a>
+      </li>
+      <li class="sec-nav-item">
+        <a class="sec-nav-link px-4" href="#">Partners</a>
+      </li>
+      <li class="sec-nav-item-btn">
+        <button class="sec-nav-link-btn px-4">Campaign</button>
+      </li>
+    </ul>
+  </div>
+
+        <div id="header">
+          <div id="header-nav-bar">
+            <nav id="burguer">
+              <div id="mobile-menu">
+                <img id= "menu-burguer" src="./capstoneimgs/burguer.svg" alt="" class="menu-icon p-3">
+                <img id= "close-menu" src="./capstoneimgs/x-button-icon-20.jpg" alt="" class="close-menu-icon p-3">
+              </div>
+              <div id="menu-container">
+                <ul class="menu">
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center text-danger" >
+                    <a href="index.html">Home</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center"> 
+                    <a href="#program">Program</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center">
+                    <a href="#join">Join</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center">
+                    <a href="#sponsor">Sponsor</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center">
+                    <a href="#news">News</a>
+                  </li>
+              </ul>
+              </div> 
+            </nav>
+          </div>                     
+        </div>
+  </section>
+
+
+      <section id="headline">
+        <p class="headline-hello">"Hello! World Music Fans!"</p>
+          <h1 class="headline-title">INTI RAYMI MUSIC FESTIVAL 2023</h1>
+        <p class="headline-p">Inti Raymi Music Festival brings together some of the most recognized musicians in Ecuador to perform for you all in the Mitad del Mundo City's Central Plaza. Come experience this celebration of thousands of years of tradition in cultural heritage.</p>
+        <p class="contact">Please contact us per email for any further questions 
+          about Inti Raymi Music Festival 2023!</p>
+        <address>intiraymifestivalecuador@somemail.not</address>
+      </section>
+    </header>
+    <main>
+      <section id="creative">
```

Comment:
- Kindly add a gray borders or hr elements to the marked sections below like on the design guideline.
- You used white background for sections but that needs to be gray. Kindly fix the background colors like I marked below.
<img width="916" alt="Screenshot 2023-03-18 at 12 04 53 PM" src="https://user-images.githubusercontent.com/25887253/226096398-adfaa1ec-b945-4bd4-a941-01c09291a421.png">